lib/locales/es.ts

```diff
@@ -261,7 +261,7 @@ export const locale: Locale = {
     clement_i_pope: 'San Clemente I, papa y mártir',
     clotilde_of_burgundy: 'Santa Clotilde',
     columban_of_luxeuil_abbot: 'San Columbano, Abad',
-    commemoration_of_all_the_faithful_departed: 'Todas las almas',
+    commemoration_of_all_the_faithful_departed: 'Conmemoración de todos los fieles difuntos',
```

Comment:
@miguelms95, as we recently started adding source comments, we need now to add a source for all changes. I have found [here](https://www.conferenciaepiscopal.es/comisiones/liturgia/) some Spanish Ordos, so I have added one of them as the source of this celebration name.

```suggestion
    commemoration_of_all_the_faithful_departed: 'Conmemoración de todos los fieles difuntos',  // src: https://www.conferenciaepiscopal.es/wp-content/uploads/2022/11/CLP-y-salmos-responsoriales-2022-2023_internet.pdf#page=342
```